config/routes.rb

```diff
@@ -16,6 +16,9 @@
   if Rails.env.development?
     mount LetterOpenerWeb::Engine, at: "/letter_opener"
   end
-
+  namespace :admin do
+    get "dashboard"
```

Comment:
Neizmanto dubultiekavas, tev visur izmantojas :symbol, tad izmanto to. Vai visur izmanto dubultiekavas. IZveilies kaut ko vienu.